examples/rust-dataflow/operator/src/main.rs

```diff
@@ -0,0 +1,47 @@
+use dora_node_api::{self, dora_core::config::DataId, DoraNode, Event, IntoArrow};
+use eyre::Context;
+
+fn main() -> eyre::Result<()> {
+    println!("hello");
+
+    let status_output = DataId::from("status".to_owned());
+    let (mut node, mut events) = DoraNode::init_from_env()?;
+
+    let mut ticks = 0;
+    while let Some(event) = events.recv() {
+        match event {
+            Event::Input { id, metadata, data } => match id.as_ref() {
+                "tick" => {
+                    ticks += 1;
+                }
+                "random" => {
+                    let value = u64::try_from(&data).context("unexpected data type")?;
+
+                    let output = format!(
+                        "operator received random value {value:#x} after {} ticks",
+                        ticks
+                    );
+                    node.send_output(
+                        status_output.clone(),
+                        metadata.parameters,
+                        output.into_arrow(),
+                    )?;
+                }
+                other => eprintln!("ignoring unexpected input {other}"),
+            },
+            Event::Stop => {} // TODO: should we stop the node here?
```

Comment:
We don't need to do a manual `break` when receiving `Event::Stop`. The daemon closes all event streams automatically when forwarding a `Stop` event: https://github.com/dora-rs/dora/blob/7761a614da17a661af39e7eecec9c88c68738e39/binaries/daemon/src/lib.rs#L1435

So the `events.recv()` call will soon return `None`. The `Stop` event is only sent to the nodes to enable them to react to manual stops in a specific way, if required.

```suggestion
            Event::Stop => {}
```